data/10-default.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "action": "install",
+    "serial": 20180113,
```

Comment:
Nitpick: I think the format in the design document was YYYYMMDDXX, where XX is a counter in case multiple changes are made in one day (like, one commit which adds multiple entries to the file). Even though it doesn’t strictly matter in this case, it’s probably best to set the right precedent. Change this to `2018011300`.

Reply:
Makes sense, I'll do that now.

Reply:
done